components/cosmetic_filters/resources/data/element_picker.ts

```diff
@@ -348,7 +349,7 @@ const attachElementPicker = () => {
     'box-shadow: none',
     'color-scheme: light dark',
     'display: block',
-    'height: 100%',
+    'height: 100lvh',
```

Comment:
Hey, does this work with only this `height: 100lvh` change? I'm struggling to see why any other changes are necessary?